### PR TITLE
Brain pediatrics updates

### PR DESCRIPTION
**Pull request by ssettipalli**
Changes/clean up related to Brain pediatrics.
Email templates are updated to reflect new styling changes.

### CODE REVIEW

File: study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/BrainConsentVersion2.java [hunk 1]
```diff
@@ -125,8 +125,10 @@ public void run(Handle handle) {
 
         //remove consent icons
         List<Long> iconIds = helper.findSectionIconIdByActivity(activityId);
-        helper.deleteActivityIcons(Set.copyOf(iconIds));
-        LOG.info("deleted form section icons for consent");
+        if (!iconIds.isEmpty()) {
```
Comment (ssettipalli) [inline study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/BrainConsentVersion2.java:135]: change to make it backward compatible if we ever re-initialize the study in lower regions.

File: study-builder/studies/brain/release.conf [hunk 2]
```diff
@@ -276,6 +276,21 @@
       "blocks": [
         {
           "componentType": "MAILING_ADDRESS",
+          "titleTemplate": {
```
Comment (ssettipalli) [inline study-builder/studies/brain/release.conf:279]: just to keep UpdateTemplatesInPlace happy because it compares titleTemplates from conf.

**Review by yufengwng — DISMISSED** [review]

File: study-builder/studies/brain/emails/about_you_completed.html [hunk 3]
```diff
@@ -113,27 +113,152 @@
...
     </td>
   </tr>
 </table>
+</body>
+
+</html>
+<html>
+<meta name="format-detection" content="telephone=no,date=no,address=no,email=no,url=no" />
+
+<head>
```
Comment (yufengwng) [inline study-builder/studies/brain/emails/about_you_completed.html]: What's going on here? Why are there two `html` tags in the same file?

Comment (ssettipalli) [reply study-builder/studies/brain/emails/about_you_completed.html]: looks like copy paste error. cleaned up

**Review by yufengwng — APPROVED** [review]

**Pull request merged**